### PR TITLE
Added a prerequisite

### PR DESCRIPTION
**Pull request by n-boshnakov**
**How to categorize this PR?**

/area documentation
/kind enhancement

**What this PR does / why we need it**:
This PR adds a prerequisite to the guide.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```doc developer
NONE
```

### CODE REVIEW

File: docs/development/getting_started_locally.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ The Gardener components, however, will be run as regular processes on your machi
     ```
 
     and reload the terminal.
+- Clone the Gardener repository from GitHub.
+  ```
+  git clone https://github.com/gardener/gardener.git
```
Comment (ialidzhikov) [inline docs/development/getting_started_locally.md]: This prerequisite is already specified in https://github.com/gardener/gardener/blob/master/docs/development/local_setup.md#get-the-sources (along with quite other local setup related tools). IMO it does not make sense to have the same prerequisite in multiple places.

Comment (n-boshnakov) [reply docs/development/getting_started_locally.md]: Thank you for your response

Comment (n-boshnakov) [reply docs/development/getting_started_locally.md]: After reviewing both documents, I have decided that while there is no need to add a duplicate prerequisite, there should be a link to the Local Setup guide. Thus there will be no duplication of prerequisites and the user will know that they need to complete the Local Setup guide before attempting to follow the Getting Started Locally tutorial.

**Comment by rfranzke** [comment]
Does it make sense to include this in `docs/deployment/getting_started_locally.md` as well?

File: docs/deployment/getting_started_locally.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ Based on [Skaffold](https://skaffold.dev/), the container images for all require
 
 ## Prerequisites
 
+- Make sure that you have followed the [Local Setup guide](https://github.com/gardener/gardener/blob/master/docs/development/local_setup.md) up until the [Get the sources](https://github.com/gardener/gardener/blob/master/docs/development/local_setup.md#get-the-sources) step.
```
Comment (ialidzhikov) [inline docs/deployment/getting_started_locally.md]: According to @dimitar-kostadinov , absolute links pointing to the same repo should be avoided - ref https://github.com/gardener/gardener/pull/5894#discussion_r863525721.

You could use relative links instead.

File: docs/development/getting_started_locally.md [hunk 3]
```diff
@@ -12,6 +12,7 @@ The Gardener components, however, will be run as regular processes on your machi
 
 ## Prerequisites
 
+- Make sure that you have followed the [Local Setup guide](https://github.com/gardener/gardener/blob/master/docs/development/local_setup.md) up until the [Get the sources](https://github.com/gardener/gardener/blob/master/docs/development/local_setup.md#get-the-sources) step.
```
Comment (ialidzhikov) [inline docs/development/getting_started_locally.md]: Same.

**Review by ialidzhikov — COMMENTED** [review]
/lgtm

**Pull request merged**